packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -63,19 +63,25 @@ if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryRespon
 				var documentEncodedString = entry.*::Document.toString();
 				var decoded = FileUtil.decode(documentEncodedString);
 				var decodedAsString = Packages.java.lang.String(decoded);
-				
+
+        if (!decodedAsString) {
+          var errorMessage = 'Error with decoded document';
```

Comment:
nit: use `const` instead of `var` when possible, and `let` when it needs to be an actual variable value.